addons/website_blog/views/snippets/s_latest_posts.xml

```diff
@@ -0,0 +1,114 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<!-- Snippets menu -->
+<template id="website_blog" inherit_id="website.snippets" name="Snippet Blog">
+    <xpath expr="//div[@id='snippet_feature']/div[hasclass('o_panel_body')]/t" position="before">
+        <t t-snippet="website_blog.s_latest_posts" t-thumbnail="/website_blog/static/src/img/s_latest_posts.jpg"/>
+        <t t-snippet="website_blog.s_latest_posts_big_picture" t-thumbnail="/website_blog/static/src/img/s_latest_posts_big_picture.jpg"/>
+    </xpath>
+</template>
```

Comment:
Does that really make sense though? We could rename snippets.xml it to snippets_menu.xml (since now it doesn't contain any snippets, but only the contents of the menu), and have snippets inject themselves into it like they do for pretty much everything else (new options, assets), this way snippets become actually self-contained. This way when you're in a snippet file, you could also see at a glance which options are available on the snippet. Just a thought.

Reply:
No, it would be a nightmare to see and change the snippets order in the menu. So best to keep snippets.xml as a file for snippet position and common snippet options.